azext_iot/digitaltwins/params.py

```diff
@@ -234,7 +235,8 @@ def load_digitaltwins_arguments(self, _):
             "properties",
             options_list=["--properties", "-p"],
             help="Initial property values for instantiating a digital twin or related components. "
-            "Provide file path or inline JSON.",
+            "Provide file path or inline JSON. Properties are required for twins that contain components, "
+            "at the minimum you must provide an object for each component that contains an empty $metadata object.",
```

Comment:
nit, I think the second line could be slightly clearer like 

`at the minimum you must provide an empty $metadata object for each component.`